Content.Shared/_RMC14/Xenonids/Construction/Nest/XenoNestSystem.cs

```diff
@@ -244,7 +244,7 @@ private void OnNestedInteractionAttempt(Entity<XenoNestedComponent> ent, ref Int
     private void TryStartNesting(EntityUid user, Entity<XenoNestSurfaceComponent> surface, EntityUid victim)
     {
         if (GetNestDirection(surface, victim) is not { } direction ||
-            !CanNestPopup(user, victim, surface, direction))
+            !CanNestPopup(user, victim, surface, direction) || !HasComp<XenoComponent>(user))
```

Comment:
This should go before the other checks so the marine doesn't even get a popup.

Reply:
Thanks, done!